pkg/apf/processor.go

```diff
@@ -42,7 +42,8 @@ func Process(data []byte, session *LMESession) bytes.Buffer {
 		break
 	case APF_CHANNEL_CLOSE: // (97) Intel AMT is closing this channel, we need to disconnect the LMS TCP connection
 		log.Debug("received APF_CHANNEL_CLOSE")
-		dataToSend = ProcessChannelClose(data, session)
+		// dataToSend =
+		ProcessChannelClose(data, session)
```

Comment:
Why are we keeping this line in the code if it is commented out?

Reply:
removed